File: objc/Project3/Practice1/Views/DetailViewController.m
//
//  DetailViewController.m
//  Practice1
//
//  Created by Jinwoo Kim on 11/18/20.
//

#import "DetailViewController.h"

@interface DetailViewController ()

@end

@implementation DetailViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do view setup here.
}

- (void)imageSelected:(NSString*)name {
    [self.imageView setImage:[NSImage imageNamed:name]];
}

@end

File: swift/Project1/Project1/DetailViewController.swift
//
//  DetailViewController.swift
//  Project1
//
//  Created by Jinwoo Kim on 11/18/20.
//

import Cocoa

class DetailViewController: NSViewController {
    @IBOutlet var imageView: NSImageView!
    
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do view setup here.
    }
    
    func imageSelected(name: String) {
        imageView.image = NSImage(named: name)
    }
}
